config.go

```diff
@@ -92,7 +92,8 @@ func (c *Config) setFlags(fs *flag.FlagSet) {
 func (c *Config) loadConfig() error {
 	fileByte, err := ioutil.ReadFile(c.ConfigPath)
 	if err != nil {
-		return err
+		log.Warnf("%s not found. Using all defaults", c.ConfigPath)
```

Comment:
Can do this or place logic around `len(c.ConfigPath)` and if it's `> 0` then we try to load config. I'm open to both use cases.